linklib/PET.MLC

```diff
@@ -1,11 +1,26 @@
-	TITLE 'PET - PRINT ELAPSED TIME IN SECONDS SINCE LAST CALL'
+**********************************************************************
+* z390 - Mainframe assembler emulator and run-time engine
+* Copyright (C) 2021 z390 Assembler LLC
 *
+* This file is part of z390.
 *
+* z390 is free software; you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation; either version 2 of the License, or
+* (at your option) any later version.
+* z390 is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License 
+* along with this program; if not, see https://www.gnu.org/licenses.
+**********************************************************************
 * 04/08/85 CODED
 * 04/28/85 ADD DISPLAY OF DATE, TIME, INSTRUCTION COUNTER, INS/SEC
 * 01/01/87 CHANGE SVC 16 TO USE CONTENTS OF R1 FOR COUNT IN R2.0
 * 10/06/09 UPDATE TO RUN ON Z390
-*
+**********************************************************************
```

Comment:
Consider moving the deleted TITLE 'PET - PRINT ... CALL' line below this line of asterisks.

Reply:
Not sure what I did with the title. Added back as requested. Fixed